PaddleAudio/examples/dcase2021_task1b/video_only_lp.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import glob
+
+import numpy as np
+import paddle
+import paddle.nn.functional as F
+from PIL import Image
+from sklearn.linear_model import LogisticRegression
+from tqdm import tqdm
+from utils import (build_rn50_model, build_rn101_model, build_vit_model,
+                   tokenize, transform)
+
+image_files = glob.glob('images/*.png')
```

Comment:
These code should put into
`if __name__ == "__main__"` code block, since this is a formal way to organize python code

Reply:
fixed.